docs/advanced_topics_sharding.rst

```diff
@@ -17,50 +17,61 @@ The parameter also ensures that all pipelines return the same number of batches,
 is divisible by the batch size but other batches are bigger by one sample. This process pads every
 shard to the same size, which is a multiple of the batch size.
 
+Framework iterator configuration
+--------------------------------
+
 DALI is used in the Deep Learning Frameworks through dedicated iterators, and these iterators need
 to be aware of this padding and other reader properties.
 
 Here are the iterator options:
 
-- ``fill_last_batch`` – Determines whether the last batch should be full, regardless of whether
-  the shard size is divisible by the batch size.
 - | ``reader_name`` - Allows you to provide the name of the reader that drives the iterator and
    provides the necessary parameters.
 
   .. note::
-    We recommend that you use this option. With this option, the next two options are excluded and
-    cannot be used.
+    We recommend that you use this option. With this option, the next two options
+    (``size`` and ``last_batch_padded``) are obtained autmatically from pipeline configuration
+    and should not be provided manually for the iterator.
 
   | This option is more flexible and accurate and takes into account that shard size for a pipeline
     can differ between epochs when the shards are rotated.
-- ``size``: Displays the size of the shard for an iterator or, if there is more than one shard,
+- ``size``: Provides the size of the shard for an iterator or, if there is more than one shard,
   the sum of all shard sizes for all wrapped pipelines.
 - | ``last_batch_padded``: Determines whether the tail of the data consists of data from the next
     shard (``False``) or is duplicated dummy data (``True``).
   | It is applicable when the shard size is not a multiple of the batch size,
-- | ``last_batch_policy`` - Determines the police about the last batch when the shard size is not
+- | ``last_batch_policy`` - Determines the policy about the last batch when the shard size is not
     divisible by the batch size.
-  | Only partially filled with the data or dropped entirely if it. The possible options are:
+  | It affects batches only partially filled with the data. It is controlled by the
+    :meth:`~nvidia.dali.plugin.base_iterator.LastBatchPolicy` enum.
```

Comment:
```suggestion
  | It affects only the batches partially filled with data. See :meth:`~nvidia.dali.plugin.base_iterator.LastBatchPolicy` enum for possible values.
```
An option is not controlled by its type.

Reply:
done